src/makeCalculator.js

```diff
@@ -4,7 +4,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculate = {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+    },
+
+    subtract(number) {
+      this.result -= number;
+    },
+
+    multiply(number) {
+      this.result *= number;
+    },
+
+    divide(number) {
+      if (number === 0) {
+        console.log('Cannot divide by zero');
+      }
+
+      this.result /= number;
```

Comment:
The division operation should not proceed if the number is zero to avoid a divide-by-zero error. The current implementation logs a message but still performs the division. Consider returning early from the method if the number is zero.